proxylib/cassandra/cassandraparser_test.go

```diff
@@ -0,0 +1,282 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cassandra
+
+import (
+	"encoding/hex"
+	"testing"
+
+	// "github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/proxylib/accesslog"
+	"github.com/cilium/cilium/proxylib/proxylib"
+	"github.com/cilium/cilium/proxylib/test"
+
+	// log "github.com/sirupsen/logrus"
+	. "gopkg.in/check.v1"
+)
+
+// Hook up gocheck into the "go test" runner.
+func Test(t *testing.T) {
+	// logging.ToggleDebugLogs(true)
```

Comment:
Leftover? you can use `-v` in go check.

for example if I run `go test -check.f "Test.*addCiliumNetworkPolicyV2" -v` inside `daemon/` it will only test the test `addCiliumNetworkPolicyV2`.

Reply:
`-v` does not trigger debug logging though. Would be handy if we could simply issue `-debug`.